configs/google-vrp/envoy-edge.yaml

```
@@ -39,36 +39,37 @@ static_resources:
           "@type": type.googleapis.com/envoy.extensions.transport_sockets.tls.v3.DownstreamTlsContext
           common_tls_context:
             tls_certificates:
-            - certificate_chain: {filename: "certs/servercert.pem"}
-              private_key: {filename: "certs/serverkey.pem"}
+            - certificate_chain: { filename: "certs/servercert.pem" }
+              private_key: { filename: "certs/serverkey.pem" }
```

mho is that this is not really an improvement - ie its not a template var or similar

the better fix would be to use proper yaml for these - eg

```suggestion
              private_key:
                filename: "certs/serverkey.pem"
```